pdr_backend/lake/etl.py

```diff
@@ -80,5 +129,6 @@ def update_bronze_pdr_predictions(self):
         print("update_bronze_pdr_predictions - Update bronze_pdr_predictions table.")
         data = get_bronze_pdr_predictions_data_with_SQL(self.ppss)
         TableRegistry().get_table(bronze_pdr_predictions_table_name).append_to_storage(
-            data
+            data,
+            build_mode=True,
```

Comment:
1. While running ETL, append_to_db(build_mode = True).
2. After successfully building all tables, dump from tables => CSV (so that we can guarantee no errors)
3. Then, finally, move all records from build_tables => final_tables

These 3 steps should guarantee a clean build, clean CSV, and no hanging records.